pages.es/common/cd.md

```diff
@@ -0,0 +1,19 @@
+# cd
+
+> Cambiar el directorio de trabajo.
+
+- Accede al directorio especificado:
+
+`cd {{ruta/al/directorio}}`
+
+- Accede al directorio *home* del usuario actual:
```

Comment:
I'd rather remove the italic marks around `home`.

Reply:
I write "home" with italic marks because in Spanish is necessary to mark in italic the words in another language (it is a rule of the official Spanish). As `home` is known as `home` in whatever language, I think that it should be in italic at least.

Is there any reason why you think that despite this rule they should be retired?

Reply:
It's fine then. Don't you translate home/ directory's name though?

Reply:
I think this is referring to the home directory here @zdroid, which is a special concept / name.

Reply:
Yes, I am referring to `home`. I speak from Spanish-speaker perspective and trying to follow as possible the grammar-spelling rules. In the case of `home` directory in all the Linux distributions I have used in Spanish, it is always `home`.

Reply:
Okay, so we keep the italicized version?

Reply:
I am waiting your decision about this kind of stuff.

Reply:
I say that we keep the italics here.